controllers/providers/assistant/gslb.go

```diff
@@ -189,32 +189,24 @@ func (r *GslbLoggerAssistant) InspectTXTThreshold(fqdn string, fakeDNSEnabled bo
 		log.Info().Msgf("Error contacting EdgeDNS server (%s) for TXT split brain record: (%s)", ns, err)
 		return err
 	}
-	var timestamp string
 	if len(txt.Answer) > 0 {
 		if t, ok := txt.Answer[0].(*dns.TXT); ok {
-			log.Info().Msgf("Split brain TXT raw record: %s", t.String())
-			timestamp = strings.Split(t.String(), "\t")[4]
+			timestamp := strings.Split(t.String(), "\t")[4]
 			timestamp = strings.Trim(timestamp, "\"") // Otherwise time.Parse() will miserably fail
-		}
-	}
-
-	if len(timestamp) > 0 {
-		log.Info().Msgf("Split brain TXT raw time stamp: %s", timestamp)
-		timeFromTXT, err := time.Parse("2006-01-02T15:04:05", timestamp)
-		if err != nil {
-			return err
-		}
-
-		log.Info().Msgf("Split brain TXT parsed time stamp: %s", timeFromTXT)
-		now := time.Now().UTC()
-
-		diff := now.Sub(timeFromTXT)
-		log.Info().Msgf("Split brain TXT time diff: %s", diff)
+			timeFromTXT, err := time.Parse("2006-01-02T15:04:05", timestamp)
+			if err != nil {
+				log.Info().Msgf("Split brain TXT [raw record: %s, raw timestamp: %s]", t.String(), timestamp)
```

Comment:
Maybe I am missing intention here but the message does not indicate that `time.Parse` has failed.